gulpfile.js

```diff
@@ -894,6 +896,30 @@ function dist() {
         if (argv.fortesting) {
           return enableLocalTesting(minifiedRuntimeTarget);
         }
+      }).then(() => {
+        /* Copy v0.js to v0-nomodule.js and fix and
```

Comment:
If this functionality is moved to it's own discreet file it might be easier to invoke the code with a function.

Also, this explanation can be expanded in a separate file. For instance... why does a `<script type=module` require no top level references to `this`? The explanation will help other developers (and yourself!) if this code needs to be modified later.

Reply:
Moved to a different file